index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <main>
+   <header>
+    <div class="oluchi"><a href="#">Oluchi Theresa</a></div>
+    <div class="port"><a href="#">Portfolio</a></div>
+    <div class="about"><a href="#">About</a></div>
+    <div class="contact"><a href="#">Contact</a></div>
+   </header>
```

Comment:
- [ ] Great job using `<header>` tag 👌, but as I mentioned ⬆ this section should only contain the logo and menu-icon because we are applying the mobile version first 💯